sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -964,6 +982,117 @@ def test_child_class_without_env_vars(self):
       actual = pcoll | base.RunInference(FakeModelHandlerNoEnvVars())
       assert_that(actual, equal_to(expected), label='assert:inferences')
 
+  def test_model_manager_loads_shared_model(self):
```

Comment:
How about we add a test that  existing model references are still valid after we loaded more models than max_model ?